examples/pgai-semantic-search/requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg2-binary>=2.9.9
+requests>=2.31.0
```

Comment:
nit, missing end of file new line
(not sure if it's  something standard in `requirements.txt` though)